src/web/middleware/sentry.rs

```diff
@@ -147,6 +149,11 @@ where
                 Some(e) => {
                     let apie: Option<&ApiError> = e.as_error();
                     if let Some(apie) = apie {
+                        if apie.is_conflict() {
+                            if let Some(state) = sresp.request().app_data::<Data<ServerState>>() {
+                                Metrics::from(state.as_ref()).incr("db.conflict");
+                            };
+                        }
```

Comment:
I imagine we will have other errors which we want to always increment a metric for. We might want to generalize this so that this hook is implemented on the error. We could make an `on_response(&ServerState)` hook for `ApiError` and then implement this logic there instead of directly in the sentry middleware.

Reply:
not a bad idea. thanks!